src/Components/RelatedProducts/RPComponents/Carousel/Carousel.jsx

```diff
@@ -23,13 +21,16 @@ class Carousel extends Component {
   }
 
   componentDidMount() {
-    fetch(`http://52.26.193.201:3000/products/${this.props.currentProduct}/related`)
+    const { mainProductId } = this.props;
```

Comment:
is this pulling data from redux?

Reply:
no this is fetching from the server direct to this component